tests/cross/src/main/scala/tests/pc/BaseTypeDefinitionSuite.scala

```diff
@@ -0,0 +1,65 @@
+package tests.pc
+
+import tests.BasePCSuite
+import scala.concurrent.duration.Duration
+import scala.meta.internal.metals.CompilerOffsetParams
+import collection.JavaConverters._
+import org.eclipse.{lsp4j => l}
+
+class BaseTypeDefinitionSuite extends BasePCSuite {
+
+  override def beforeAll(): Unit = {
+    indexJDK()
+    indexScalaLibrary()
+  }
+
+  def locationFromString(str: String): l.Location = {
+    def positionFromString(str: String): l.Position = {
+      val ints = str.split(":").map(_.toInt)
+      new l.Position(ints.head, ints.last)
+    }
+
+    val split = str.split("""/\\""")
+    assertEquals(split.length, 3, "Incorrect location string format")
+    val (uri, rest) = (split.head, split.tail)
+    val range =
+      new l.Range(positionFromString(rest.head), positionFromString(rest.last))
+    new l.Location(uri, range)
+  }
+
+  def locationToString(loc: l.Location): String = {
+    val (uri, start, end) =
+      (loc.getUri, loc.getRange.getStart, loc.getRange.getEnd)
+    s"|$uri/\\${start.getLine}:${start.getCharacter}/\\${end.getLine}:${end.getCharacter}|"
+  }
+
+  def check(
```

Comment:
we can change it a bit to make the call site a bit nicer: 
```scala 
def check(name:String)(code:String, expected:String,duration:Duration=(...))
```

now, we call it like:
```scala
check("foo")(
  code = ???,
  expected = ???
)
```

Reply:
Fixed.